index.js

```diff
@@ -100,4 +101,12 @@ module.exports = {
       );
     }
   },
+  onEnd: async () => {
```

Comment:
This cleans up some transient files that are generated during the plugin's run.
Useful when running locally via `netlify build`